src/main/java/com/stripe/model/ExternalAccountCollection.java

```diff
@@ -1,63 +1,55 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
 package com.stripe.model;
 
 import com.stripe.Stripe;
 import com.stripe.exception.StripeException;
 import com.stripe.net.ApiResource;
 import com.stripe.net.RequestOptions;
-
 import java.util.Map;
 
 public class ExternalAccountCollection extends StripeCollection<ExternalAccount> {
-  // <editor-fold desc="all">
-  @Deprecated
-  public ExternalAccountCollection all(Map<String, Object> params) throws StripeException {
-    return list(params, null);
-  }
-
-  @Deprecated
-  public ExternalAccountCollection all(Map<String, Object> params, RequestOptions options)
-      throws StripeException {
-    return list(params, options);
-  }
-  // </editor-fold>
-
-  // <editor-fold desc="create">
-  public ExternalAccount create(Map<String, Object> params) throws StripeException {
-    return create(params, null);
-  }
-
-  public ExternalAccount create(Map<String, Object> params, RequestOptions options)
-      throws StripeException {
-    return ApiResource.request(ApiResource.RequestMethod.POST, String.format("%s%s",
-        Stripe.getApiBase(), this.getUrl()), params, ExternalAccount.class, options);
-  }
-  // </editor-fold>
-
-  // <editor-fold desc="list">
+  /** List external accounts for an account. */
   public ExternalAccountCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+    return list(params, (RequestOptions) null);
   }
 
+  /** List external accounts for an account. */
```

Comment:
10) List (standard): Instance method on to list sub-resource on collection already containing ID of parent resource.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/ExternalAccountCollection.java#L17-L22